### DISCUSSION TITLE
The password security of on-premise installation.

### DISCUSSION THREAD

**Discussion by abbi1229** [ankitpokhrel/jira-cli#217] [Q&A]
For on-premise installation. We couldn't create a token by jira API.  (Before 8.14 version)
So, we need to save the password without encoded in ``./bashrc``.
It might be too easy to steal by other user.
For me, I use ``set JIRA_API_TOKEN ****`` and ``unset JIRA_API_TOKEN`` every time when using JiraCLI.

Is there any safer way to hide or encode the password?

**Comment by ankitpokhrel** [comment]
Hi @abbi1229, would supporting [.netrc](https://www.gnu.org/software/inetutils/manual/html_node/The-_002enetrc-file.html) solves your problem?

**Reply by abbi1229** [reply]
Hi @ankitpokhrel , 
It would be helpful if the password is not saved in a file without encoding.

**Comment by ankitpokhrel** [comment]
Support for .netrc and keychain is available in the new release [v1.0.0-beta](https://github.com/ankitpokhrel/jira-cli/releases/tag/v1.0.0-beta)
